server/modules/allocations.js

```diff
@@ -65,10 +65,10 @@ export async function findPartnerById(partnerId) {
  * @returns {Promise} - Promise which resolves to a list of placements
  * from the past numberOfDays provided, or throws an error if unsuccessful
  */
-export const fetchNewPlacements = async (status, numberOfDays = 1) => {
-  const response = await axios.get(`api/v1/placements?status=${status}`);
+export const fetchNewPlacements = async (status) => {
+  const response = await axios.get(`http://localhost:8000/api/v1/auto/placements?status=${status}`);
```

Comment:
Would be nice to set `BASE_URL=http://localhost:8000` as an environment variable, then make use of it here and other places, than hardcoding the values. This would ensure maintainability

Reply:
Working on this.